templates/coderd.yaml

```diff
@@ -132,20 +111,15 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: DEVURL_HOST
-              value: {{ include "movedValue" (dict "Values" .Values "Key" "coderd.devurlsHost") }}
+              value: {{ .Values.coderd.devurlsHost | quote }}
             - name: POD_IP
               valueFrom:
                 fieldRef:
                   fieldPath: status.podIP
             - name: VERBOSE
               value: "true"
-            {{- if and (eq (include "coder.hasNginxIngress" .) "true") (eq (merge .Values dict | dig "coderd" "serviceNext" false) false) (merge .Values dict | dig "ingress" "enable" true) (merge .Values dict | dig "ingress" "useDefault" true) }}
```

Comment:
Shouldn't this still be on if `ingress.enabled`?

Reply:
I don't think we care if they're using our included ingress or not. Letting them trust their own headers will be good for some customers since they have other teams running WAFs/proxies/etc.

Reply:
if the ingress is enabled, I guess we force trust proxy IP to on - is that the desired behavior?

Reply:
seems probably safer to just require the user to configure trustProxyIP themselves, even if they've enabled the ingress, so maybe that's the move here?